pytest issue

=========================== short test summary info ============================
FAILED tests/llm/test_dr_research.py::test_research_generates_findings - UnboundLocalError: cannot access local variable 'os' where it is not associated with a value
FAILED tests/llm/test_dr_scope.py::test_scope_returns_brief - UnboundLocalError: cannot access local variable 'os' where it is not associated with a value
FAILED tests/llm/test_dr_write.py::test_write_composes_report - UnboundLocalError: cannot access local variable 'os' where it is not associated with a value
FAILED tests/llm/test_dr_write.py::test_run_pipeline - UnboundLocalError: cannot access local variable 'os' where it is not associated with a value
======== 4 failed, 8 passed, 1 skipped, 37 warnings in 79.95s (0:01:19) ========

--- a/omicverse/llm/dr/research_manager.py
+++ b/omicverse/llm/dr/research_manager.py
@@ -83,8 +83,6 @@ class ResearchManager:
                     EmbedWebRetriever = None  # type: ignore
 
                 if vs_flag == "web":
-                    import os
-
                     backend = "tavily" if os.getenv("TAVILY_API_KEY") else ("brave" if os.getenv("BRAVE_API_KEY") else "duckduckgo")
                 elif vs_flag in {"web:tavily", "web:embed:tavily"}:
                     backend = "tavily"

